oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -156,7 +156,15 @@ storage:
 receiver-sharing-server:
   selector: ${SW_RECEIVER_SHARING_SERVER:default}
   default:
+    host: ${SW_RECEIVER_JETTY_HOST:0.0.0.0}
```

Comment:
Don't open this, because once you did, all query and trace in http need to be reported through this port, which breaks quick start and tests.

Your change should be in the doc mostly.

Reply:
We can also set port default to 0?